server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs

```diff
@@ -63,10 +69,6 @@ public async Task<ActionResult<MistakeWebModel>> AddMistake(NewMistakeWebModel n
         public async Task<ActionResult<MistakeWebModel>> CloneMistake(Guid mistakeId)
         {
             var mistake = await _dataContext.Set<Mistake>()
-                .Include(m => m.Tips)
-                .Include(m => m.MistakeLabels)
-                    .ThenInclude(m => m.Label)
-                .Include(m => m.Repetitions)
```

Comment:
Just to make sure: after cloning, won't new mistake have empty fields?

Reply:
you were right, fixed